+nosnoc/+model/Heaviside.m

```diff
@@ -0,0 +1,46 @@
+classdef Heaviside < nosnoc.model.Base
+    properties
```

Comment:
Add short comment  (1-2 sentences) why is this not a PSS in some cases (will be in docu anyway)